### PR TITLE
chore(wallet): move bridge to its own folder

### PR DESCRIPTION
**Pull request by J-Son89**
Going to make a set of pr's to organise the wallet folder a bit better. 

This pr moves the bridge related code to its own folder which aligns better with the associated design file. 

I also updated the names of the bridge related screens to be more expressive of what is done on those pages.

Note:
I believe this pr can skip manual QA as it is rather straightforward.

### CODE REVIEW

**Comment by J-Son89** [comment]
going to skip manual QA here as this is rather trivial and the compiler should catch everything except for the screen changes. However we can verify this ourselves rather easily.
Happy to change if anyone prefers 👍 
cc @briansztamfater @mmilad75

File: src/status_im/contexts/wallet/bridge/bridge_to/style.cljs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+(ns status-im.contexts.wallet.bridge.bridge-to.style)
+
+(def content-container
+  {:padding-horizontal 8})
+
```
Comment (smohamedjavid) [inline src/status_im/contexts/wallet/bridge/bridge_to/style.cljs:5]: Extra line :)

**Review by ulisesmac — APPROVED** [review]

**Review by smohamedjavid — APPROVED** [review]

**Comment by J-Son89** [comment]
> ## 96% of end-end tests have passed
> ```
> Total executed tests: 48
> Failed tests: 1
> Expected to fail tests: 1
> Passed tests: 46
> ```
> 
> ```
> IDs of failed tests: 702777 
> ```
> 
> ```
> IDs of expected to fail tests: 703503 
> ```
> 
> ### Failed tests (1)
> Click to expand
> ### Expected to fail tests (1)
> Click to expand
> ### Passed tests (46)
> Click to expand

look okay @pavloburykh ?

File: src/status_im/contexts/wallet/bridge/confirm_transaction/view.cljs [hunk 2]
```diff
@@ -11,7 +11,7 @@
   []
   [rn/view {:style style/bridge-send-wrapper}
    [input-amount/view
```
Comment (briansztamfater) [inline src/status_im/contexts/wallet/bridge/confirm_transaction/view.cljs]: I am confused why this is not `bridge.input-amount`?

Comment (J-Son89) [reply src/status_im/contexts/wallet/bridge/confirm_transaction/view.cljs]: I must have got it backwards 🙃 

I'll adjust!

File: src/status_im/contexts/wallet/bridge/input_amount/view.cljs [hunk 3]
```diff
@@ -1,9 +1,9 @@
-(ns status-im.contexts.wallet.account.bridge.view
+(ns status-im.contexts.wallet.bridge.input-amount.view
```
Comment (briansztamfater) [inline src/status_im/contexts/wallet/bridge/input_amount/view.cljs:1]: Shouldn't this be `bridge.select-asset` or `bridge.select-token`?

Comment (J-Son89) [reply src/status_im/contexts/wallet/bridge/input_amount/view.cljs:1]: yeah sounds good to me! will adjust!

**Comment by J-Son89** [comment]
how about now @briansztamfater 🤔

**Comment by pavloburykh** [comment]
> look okay @pavloburykh ?

@J-Son89 looks perfect.

**Review by briansztamfater — APPROVED** [review]

**Comment by briansztamfater** [comment]
> how about now @briansztamfater 🤔

Not been able to test it but now looks good!

**Comment by J-Son89** [comment]
No need to test, was just wondering about the code. Thanks:)

**Pull request merged**